src/Platform.Handlers/src/Xamarin.Platform.Handlers/Platform/Android/DrawableExtensions.Android.cs

```diff
@@ -1,23 +1,22 @@
 ﻿using System;
 using Android.Graphics;
+using Xamarin.Forms;
 using AColor = Android.Graphics.Color;
 using AColorFilter = Android.Graphics.ColorFilter;
 using ADrawable = Android.Graphics.Drawables.Drawable;
+
+#if __ANDROID_29__
```

Comment:
You can get rid of all these if defs

this is only going to be API 29+ so they aren't needed

Reply:
But this DrawableExtensions class is also used by the Xamarin.Forms renderers.

Reply:
That's fine we've removed API 28 from main on Xamarin.Forms 

We should remove all `#if __ANDROID_29__` from the main branch as well